_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -929,6 +929,59 @@ public function update_data( $request ) {
 
 	}
 
+	/**
+	 * Install a plugin.
+	 *
+	 * @since 5.8.0
+	 *
+	 * @param string $slug Plugin slug.
+	 *
+	 * @return bool True if installation succeeded, error object otherwise.
+	 */
+	static function _install_plugin( $slug ) {
```

Comment:
> why the underscore in the method name?

I was mimicking the (private) `_process_onboarding()` method's nomenclature here (since `_install_plugin`, while not private, is only used by that private method). I'm perfectly fine with dropping it (especially if I move it to a different file and it becomes a public function).

Reply:
Ah, makes sense. Anyway, I still don't see why we should be prefixing with an underscore - why not just making the method `private` or `protected`? That applies to `_process_onboarding()` too.

Reply:
I think the idea is just to signal "this method is private" as part of its name (so its immediately visible at call sites). A bit like including type information in a variable name (can't think of anything really ugly now, maybe something like `counter_int` or `__super_ugly_struct_t` or `CFooBarSubClass` 😛).

Reply:
I still don't follow why prefixing with an underscore would be more immediately visible than having the method defined with `protected` or `private` visibility. After all, visibility is specified before the method name. Not to mention that if a method **should** be considered private, it is beneficial that it **should** be implemented as such, so it's not inadvertently used as a public one.